### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by vpai**
Improve our README. 📈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,96 @@
...
+
+## Installation
+
+```
+npm install @eppo/js-client-sdk
+```
+
+## Usage
+
+Initialization creates a singleton instance of Eppo's client and fetches the account's flag configuration, which is cached locally. Once initialized, assignments can be made anywhere in your app.
```
Comment (ploomiz) [inline README.md:18]: Suggested rephrase -

"Once initialized, the SDK can assign variations from anywhere in your app."

**Review by ploomiz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,11 +1,125 @@
...
+[Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool. Eppo's Javascript SDK is built to make assignments for single user client applications that run in a web browser. Before proceeding you'll need an Eppo account.
+
+## Features
+
+- Feature gates
+- Kill switches
+- Progressive rollouts
+- A/B/n experiments
+- Mutually exclusive experiments (Layers)
+- Contextual bandits
```
Comment (giorgiomartini0) [inline README.md:15]: Nit: these are upcoming but not yet available. (The former in general, the latter for the JS SDK.)

File: README.md [hunk 3]
```diff
@@ -1,11 +1,125 @@
...
+```javascript
+import * as EppoSdk from "@eppo/js-client-sdk";
+
+const eppoClient = EppoSdk.getInstance();
+const user = getCurrentUser();
+
+const variation = eppoClient.getStringAssignment('new-feature', user.id, { 
+  'country': user.country,
+  'device': user.device,
+}, false);
```
Comment (giorgiomartini0) [inline README.md:49]: The default should be a string, not `false`.

**Review by giorgiomartini0 — COMMENTED** [review]
🙌

File: README.md [hunk 4]
```diff
@@ -1,11 +1,124 @@
...
+
+Each function has the same signature, but returns the type in the function name. For booleans use `getBooleanAssignment`, which has the following signature:
+
+```javascript
+getBooleanAssignment: (
+  flagKey: string,
+  subjectKey: string,
+  subjectAttributes: Record<string, any>,
+  defaultValue: string,
+) => boolean
```
Comment (schmit) [inline README.md:71]: should this be? 
```suggestion
): boolean
```

**Review by schmit — APPROVED** [review]

**Review by giorgiomartini0 — APPROVED** [review]

**Pull request merged**